src/apps/content-editor/src/app/views/ItemList/SetRow/ToggleCell/ToggleCell.js

```diff
@@ -15,26 +18,39 @@ export const ToggleCell = memo(function ToggleCell(props) {
       if (options[0].length > 3 || options[1].length > 3) {
         return (
           <span className={cx(props.className, styles.ToggleCell)}>
-            <ToggleButton
-              name={props.name}
+            <ToggleButtonGroup
+              color="secondary"
               value={props.value}
-              offValue="&nbsp;&nbsp;&nbsp;"
-              onValue="&nbsp;&nbsp;&nbsp;"
-              onChange={props.onChange}
-            />
-            {props.field.settings.options[props.value]}
+              size="small"
+              onChange={(e, val) => {
+                if (val !== null) {
+                  props.onChange(val, props.name);
+                }
+              }}
+              onClick={(evt) => evt.stopPropagation()}
```

Comment:
Why are we stopping event propagation?

Reply:
When selecting true/false it was redirecting the user to the item page instead of selecting the toggle button. 

![bubble-link](https://user-images.githubusercontent.com/22800749/169905581-c0116b84-0b2e-453f-8479-845999869276.png)